packages/security-questions/src/SecurityQuestionStore.ts

```diff
@@ -12,22 +12,26 @@ class SecurityQuestionStore implements ISerializable {
 
   questions: string;
 
-  constructor({ nonce, shareIndex, sqPublicShare, polynomialID, questions }: SecurityQuestionStoreArgs) {
+  keyType: KeyType;
+
+  constructor({ nonce, shareIndex, sqPublicShare, polynomialID, questions, keyType }: SecurityQuestionStoreArgs) {
     this.nonce = new BN(nonce, "hex");
     this.shareIndex = new BN(shareIndex, "hex");
-    this.sqPublicShare = new PublicShare(sqPublicShare.shareIndex, sqPublicShare.shareCommitment);
+    this.keyType = keyType in KeyType ? keyType : KeyType.secp256k1;
```

Comment:
I think keyType here should be require ( no optional ) ( pass in from the tkey )
more deterministic